schema/crdb/add-ip-version-to-pools/up03.sql

```diff
@@ -0,0 +1,3 @@
+SET disallow_full_table_scans = 'off';
+UPDATE omicron.public.ip_pool SET ip_version = 'v4';
+SET disallow_full_table_scans = 'on';
```

Comment:
Not at all important, but FYI: if you use `local` you don't have to unset it.

<img width="502" height="388" alt="Image" src="https://github.com/user-attachments/assets/d3e2796f-2fe0-41a9-866b-549c1be44bac" />

Reply:
Changed in e8ff83a94